pkg/scheduler/apis/config/types.go

```diff
@@ -115,6 +117,16 @@ type KubeSchedulerProfile struct {
 	// is scheduled with this profile.
 	SchedulerName string
 
+	// PercentageOfNodesToScore is the percentage of all nodes that once found feasible
+	// for running a pod, the scheduler stops its search for more feasible nodes in
+	// the cluster. This helps improve scheduler's performance. Scheduler always tries to find
+	// at least "minFeasibleNodesToFind" feasible nodes no matter what the value of this flag is.
+	// Example: if the cluster size is 500 nodes and the value of this flag is 30,
+	// then scheduler stops finding further feasible nodes once it finds 150 feasible ones.
+	// When the value is 0, default percentage (5%--50% based on the size of the cluster) of the
```

Comment:
say "adaptative percentage" instead of "default"

Reply:
Although, in v1beta1 case, 0 means that top-level PercentageOfNodesToScore is used. That's fine with me.

Reply:
It seems that this requires API review :) @alculquicondor